smt/sampling_methods/tests/test_lhs.py

```diff
@@ -44,7 +44,7 @@ def test_expand_lhs(self):
             new = int(new_list[i])
             new_num = num + new
 
-            x_new = sampling.expand_lhs(x, new)
+            x_new = sampling.expand_lhs(x, new, method="ese")
```

Comment:
Assertions below did not change. Could you add something related to the fact that "ese" is used?